system.go

```diff
@@ -108,7 +108,7 @@ func (u *UserAgent) isBot() bool {
 		return true
 	}
 
-	if u.Browser.Name == BrowserBot {
+	if u.Browser.Name >= BrowserBot {
```

Comment:
This could break if we add more useragents that are not bots.

For new, please add an upper limit to the check and add a comment at the end of the BrowserName struct so that we remember to update this code in future.